### ISSUE TITLE
[CLI] wandb_watch='all' doesn't log parameters

### ISSUE THREAD

**Issue by andreimargeloiu** [wandb/wandb#4261]
### Describe the bug

**Context**: I'm using pytorch-lightning and wandb, investigating how the parameters evolve during training. I set ```wandb.watch(model, log='all', log_freq=10)``` and expect to see the parameters and gradients inside the UI.

**Problem**: Only the gradients are logged into the UI. The parameters aren't logged during training. [Link](https://wandb.ai/andreimargeloiu/network_initialisation/runs/13u4u8k0) to the run.

What can be the problem?

![CleanShot 2022-09-12 at 16 37 43@2x](https://user-images.githubusercontent.com/18227298/189696359-047676f6-75f4-4cfa-8bcb-41905762458e.png)

### Environment

WandB version: 0.13.3
Pytorch-lightening: 1.7.5
torch: 1.10.1

Python: 3.7.9

### Additional Context

Other people discussing this problem online:
- [Here](https://community.wandb.ai/t/wandb-watch-with-pytorch-not-logging-anything/2096/3?u=andreimargeloiu) they say that using ```forward()``` instead of ```__call__()``` fixed the problem. I'm currently using forward().

**Comment by ramit-wandb** [comment]
Hi @andreimargeloiu,

I just tested this on `pytorch-lightning==1.7.5` and `wandb==0.13.3`, and I was able to get both gradients and parameters to log though `wandb.watch(model, log='all')`, so I don't think the issue is arising from the libraries. Looking at your workspace, I noticed that the parameters do not seem to have been logged at all, so this can not be an issue rendering the parameters either.

Could you share the `debug.log` and `debug-internal.log` files associated to an affected run? That could be a good place to start looking for any errors.

Thanks,
Ramit

**Comment by kptkin** [comment]
Hi @andreimargeloiu and @icedpanda, 
Thanks for letting us know, sorry for the inconvenience, we will try to reproduce locally. 
For now we introduced a recent improvement in the new release 0.13.3, could you verify that it works for you in version 0.13.2?

**Comment by icedpanda** [comment]
Hi @kptkin,

my environment:
- wandb: 0.13.2
- lighting 1.7.3
- torch: 1.10
- python: 3.9

**Comment by kptkin** [comment]
Hi @icedpanda, thanks for the information, ok so from your environment, it seems to be an unrelated issue, we are still looking into it. Will update when fix is available, sorry for the inconvenience.

**Comment by icedpanda** [comment]
@kptkin `wandb.watch` works now after a clean reinstall

**Comment by kptkin** [comment]
> @kptkin `wandb.watch` works now after a clean reinstall

Thanks for the update glad to hear it was resolved for you!
